static/js/editor.js

```diff
@@ -0,0 +1,18 @@
+var editorTitle = new Quill('#editorTitle')
+
+var editorContent = new Quill('#editorContent', {
+  theme: 'bubble',
+  modules: {
+    toolbar: [
+      ['bold', 'italic'],
+      ['link', 'image']
+    ]
+  }
+})
+
+document.querySelector('form[edit]').addEventListener('submit', event => {
+  const form = event.target
+  form.title.value = editorTitle.getContents()
```

Comment:
Just get the title from the `textContent` [like we did](https://github.com/neomad-team/neomad.org/pull/228/files#diff-b2eb8af5867e7d92dac4de3e62798caeL105)